app/models/contents.rb

```diff
@@ -0,0 +1,11 @@
+require 'ostruct'
```

Comment:
Hmmm, while I see what you were doing here, I'm not sure we need to create a separate class to handle note validations, we can just check within the relevant model method. I'm going to sleep on it before we make a final decision on that though.

Regardless, if we *did* want to add a general purpose note validator, we would probably a) want to check that the contents are not empty as well (e.g. minimum length 1 or higher) and b) make `max_length` an instance variable as opposed to a class variable so you could set it on creation if you wanted (e.g. write an initializer method for the class that defaults to 500 characters but allows you to set it if you want). This way we could use it in circumstances other than those where want a limit of 500 characters.